src/icon-loader.c

```diff
@@ -25,6 +25,18 @@ struct icon_loader {
 	struct sfdo_icon_theme *icon_theme;
 };
 
+static void
+log_handler(enum sfdo_log_level level, const char *fmt, va_list args, void *data)
+{
+	// add a prefix if the format length is reasonable
+	char buf[256];
+	if (snprintf(buf, sizeof(buf), "[sfdo] %s", fmt) < (int)sizeof(buf)) {
```

Comment:
I suggest setting prefixes like `[sfdo-desktop]` and `[sfdo-icon]`. I think the argument `data` can be used for this.